### PR TITLE
Key bindings in config now override defaults (Tuxemon#767)

### PR DESCRIPTION
**Pull request by austintian03**
This fix reverses the key/value pairs of the `controls` section of the config. Now the key is the pygame internal action, and the value is a comma separated list of strings for your keyboard buttons. Since the keys in the config section never get deleted or modified, you can modify the controls without `generate_default_config()` always adding back the default controls.

### CODE REVIEW

**Comment by Qiangong2** [comment]
How would this work in the Android version of the game? Do the controls still work? Since it uses an overlay, this would affect the configuration wouldn't it?

**Comment by bitcraft** [comment]
@Qiangong2 is is not related to the android build.  controls are no longer bound to the keyboard and multiple virtual controllers are available to use at any time.  the overlay needs to be fixed though.

File: tuxemon/core/config.py [hunk 1]
```diff
@@ -126,18 +126,20 @@ def get_custom_pygame_keyboard_controls(cfg):
...
-        key = key.lower() if len(key) == 1 else key.upper()
-        value = value.upper()
-        pygame_value = getattr(pygame.locals, "K_" + key, None)
-        button_value = getattr(buttons, value, None)
-        event_value = getattr(events, value, None)
-        if pygame_value is not None and button_value is not None:
-            custom_controls[pygame_value] = button_value
-        elif pygame_value is not None and event_value is not None:
-            custom_controls[pygame_value] = event_value
+    #custom_controls = PygameKeyboardInput.default_input_map.copy()
```
Comment (bitcraft) [inline tuxemon/core/config.py:129]: please remove this comment

Comment (bitcraft) [inline tuxemon/core/config.py:129]: please remove this comment

**Comment by Qiangong2** [comment]
@austintian03 Please remove the comment so that we may get this merged in.

**Comment by bitcraft** [comment]
Thank you @austintian03 !

**Pull request merged**